### ISSUE TITLE
Realize first draft of intro scene in Unity

### ISSUE THREAD

**Issue by Kittyfisto** [Kittyfisto/Taleon#16]
The intro (as described in 'Chapter 0 - Intro.txt') shall be realized in Unity in the 'Demo' scene:

The player's main ship shall be placed inside the hangar. Light sources must be present *inside* the hangar.

A proposed cut scene shall be developed that includes camera cuts from various angles, as described in the script.

Launching the 'Demo' scene shall automatically start this cut scene. Once the animation is finished, the current main camera shall simply remain stationary.

**Comment by Kittyfisto** [comment]
Implemented half of the proposed camera angles. All further cuts are postponed until the combat prototype is in satisfactory condition.
